backend/packages/Upgrade/src/api/repositories/ExperimentRepository.ts

```diff
@@ -78,21 +78,22 @@ export class ExperimentRepository extends Repository<Experiment> {
   }
 
   public async getValidExperiments(context: string): Promise<Experiment[]> {
+    const whereExperimentsClause =
```

Comment:
Make these variables global as they are used across functions.

Reply:
In getValidExperimentsWithPreview function, the whereClause also has preview experiments. So, we cant have it as global.